test-data/unit/lib-stub/enum.pyi

```diff
@@ -21,6 +21,10 @@ class Enum(metaclass=EnumMeta):
     _name_: str
     _value_: Any
 
+    # In reality, _generate_next_value_ is python3.6 only and has a different signature.
```

Comment:
Shenanigans are happening here -- I'm not sure the best way to do this.  If I use the signature (as written in `typeshed`), we end up with:

```
        @staticmethod
        def _generate_next_value_(name: str, start: int, count: int, last_values: List[Any]) -> Any: pass
```

That pulls in `staticmethod` which doesn't have a stub in `lib-stub`.  From that point, there are a bunch of different ways we can lie (e.g. just add a `self` and drop the `staticmethod`), but it seems to me like a little lie is just as harmful as a big one (maybe more because it's less likely to be noticed).

Reply:
I think this comment is all we need if in the future mypy and/or the test fixtures somehow get clever enough to notice the signature mismatch.